apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1.IntegrationTests/RetentionPolicyTest.cs

```diff
@@ -206,5 +264,12 @@ private Object CreateObject(string bucketName, string objectName)
             var bytes = Encoding.UTF8.GetBytes(text);
             return _fixture.Client.UploadObject(bucketName, objectName, "text/plain", new MemoryStream(bytes));
         }
+
+        private Object CreateObject(Object obj)
```

Comment:
This one is probably better in the Fixture. Seems like we have one already that's local to another method, but we might be able to easily make it internal.

Reply:
And ping on this comment?

Reply:
I'd probably remove the method entirely, and just inline `_fixture.Client.UploadObject(obj, new MemoryStream(_fixture.SmallContent));` unless we really need the specific text.